src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices.test.ts

```diff
@@ -0,0 +1,293 @@
+import '@testing-library/jest-dom'
+
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { h } from 'react-hyperscript-helpers'
+import { ButtonPrimary } from 'src/components/common'
+import { cloudServices } from 'src/data/gce-machines'
+import { formatUSD } from 'src/libs/utils'
+import { azureRuntime, getAzureDisk, getDisk, getJupyterRuntimeConfig } from 'src/pages/workspaces/workspace/analysis/_testData/testData'
+import { DeleteDiskChoices, DeleteEnvironment } from 'src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices'
+
+
+const renderActionButton = () => h(ButtonPrimary, {}, ['Delete'])
+
+describe('compute-modal-component', () => {
```

Comment:
Could you explain the additional describe?
I don't think we use this pattern elsewhere - wondering what your thoughts are.

Reply:
I saw the pattern in one other place, I'll just go ahead and nix it